snuba/datasets/processors/errors_processor.py

```diff
@@ -238,13 +248,22 @@ def extract_tags_custom(
     def extract_promoted_contexts(
         self,
         output: MutableMapping[str, Any],
-        contexts: Mapping[str, Any],
-        tags: Mapping[str, Any],
+        contexts: MutableMapping[str, Any],
+        tags: MutableMapping[str, Any],
     ) -> None:
         transaction_ctx = contexts.get("trace") or {}
         trace_id = transaction_ctx.get("trace_id", None)
         span_id = transaction_ctx.get("span_id", None)
 
+        replay_ctx = contexts.get("replay") or {}
+        replay_id = replay_ctx.get("replay_id", None)
+
+        if replay_id:
+            replay_id_uuid = uuid.UUID(replay_id)
+            output["replay_id"] = str(replay_id_uuid)
+            tags["replayId"] = replay_id_uuid.hex
```

Comment:
Why do you overwrite this? If the customer is passing it in, shouldn't we keep it the same?

Reply:
i'll add a check to make sure we're not overwriting as to not interfere. i did a similar thing on the transactions processor PR.